.github/workflows/ci-pr.yml

```diff
@@ -28,7 +28,7 @@ jobs:
       fail-fast: false
       matrix:
         platform: ["ubuntu-24.04"]
-        partition_id: [0, 1, 2, 3, 4, 5] # set PARTITION_TOTAL below to match
+        partition_id: [0, 1, 2, 3, 4, 5] # 6 partitions total
```

Comment:
I assume that `partition-total` below still has to match? Worth mentioning here.